### ISSUE TITLE
error when using `nextclade dataset get --verbosity` flag

### ISSUE THREAD

**Issue by kapsakcj** [nextstrain/nextclade#1428]
Am I using the `--verbosity` flag incorrectly with nextclade CLI?

This command runs fine without `--verbosity` flag ✅ :
```
$ docker run --rm -v $PWD:/data nextstrain/nextclade:3.3.0 nextclade dataset get --name="sars-cov-2" --tag="2024-02-16--04-00-32Z" -o /data/nextclade_dataset_dir/
```

Same command, but `--verbosity` flag included ❌ :
```
$ docker run --rm -v $PWD:/data nextstrain/nextclade:3.3.0 nextclade dataset get --name="sars-cov-2" --tag="2024-02-16--04-00-32Z" -o /data/nextclade_dataset_dir/ --verbosity debug
The application panicked (crashed).
Message:  Mismatch between definition and access of `verbosity`. Could not downcast to TypeId { t: 38017389458503008020766281842196327386 }, need to downcast to TypeId { t: 42966343538335219590177265727833432740 }

Location: /home/user/.cargo/registry/src/index.crates.io-6f17d22bba15001f/clap_builder-4.4.2/src/parser/error.rs:32

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
```

Same as before, but with RUST_BACKTRACE environmental variable set ❌ :
```
$ docker run --rm -v $PWD:/data -e RUST_BACKTRACE=1  nextstrain/nextclade:3.3.0 nextclade dataset get --name="sars-co
v-2" --tag="2024-02-16--04-00-32Z" -o /data/nextclade_dataset_dir/ --verbosity debug
The application panicked (crashed).
Message:  Mismatch between definition and access of `verbosity`. Could not downcast to TypeId { t: 38017389458503008020766281842196327386 }, need to downcast to TypeId { t: 42966343538335219590177265727833432740 }

Location: /home/user/.cargo/registry/src/index.crates.io-6f17d22bba15001f/clap_builder-4.4.2/src/parser/error.rs:32

  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ BACKTRACE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
   1: __libc_start_main<unknown>
      at <unknown source file>:<unknown line>

Run with COLORBT_SHOW_HIDDEN=1 environment variable to disable frame filtering.
Run with RUST_BACKTRACE=full to include source snippets.
```

**Comment by kapsakcj** [comment]
I don't think this error is specific to `nextclade dataset get`, I'm also seeing this error when using the `--verbosity` flag with `nextclade run <...>` commands.

**Comment by ivan-aksamentov** [comment]
Hi Curtis @kapsakcj, 

Thanks for reporting!

I fixed it in #1429. Will release as 3.3.1 shortly.

**Comment by kapsakcj** [comment]
awesome, thank you, Ivan!
